auth/oidc/oidc_test.go

```diff
@@ -41,6 +41,9 @@ func (s *AuthSuite) SetUpSuite(c *check.C) {
 		json.NewEncoder(w).Encode(s.jwkKeySet)
 	}))
 
+	config.Set("database:url", "127.0.0.1:27017?maxPoolSize=100")
+	config.Set("database:name", "oidc_tests")
+
```

Comment:
please call `storagev2.Reset()` to ensure that old singleton database client is destructed

Reply:
added, thanks 🥇